src/main/java/com/google/sps/servlets/CreateEventServlet.java

```diff
@@ -41,12 +41,13 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
                 .set("date", date)
                 .set("details", details)
```

Comment:
Shouldn't this also be "description" as you change it above?

Reply:
I changed the request value to match the post request attribute but I kept the variable's name. It does makes it confusing, I'll change it.

Reply:
You don't need to change if for this, but I do think in general, keeping naming consistent is nice. With this approach, it's easier to see what's coming from the frontend and what's going into the backend, but overall, I think it's just easier to name everything the same